stream.go

```diff
@@ -1101,17 +1137,17 @@ func (a *csAttempt) recvMsg(m any, payInfo *payloadInfo) (err error) {
 		}
 		a.mu.Unlock()
 	}
-	for _, sh := range a.statsHandlers {
-		sh.HandleRPC(a.ctx, &stats.InPayload{
-			Client:   true,
-			RecvTime: time.Now(),
-			Payload:  m,
-			// TODO truncate large payload.
-			Data:             payInfo.uncompressedBytes,
-			WireLength:       payInfo.compressedLength + headerLen,
-			CompressedLength: payInfo.compressedLength,
-			Length:           len(payInfo.uncompressedBytes),
-		})
+	if len(a.statsHandlers) != 0 {
```

Comment:
Why do we need this `len` check?

Reply:
Ah, we used to! Because it used to materialize the buffer to go into the `Data` field. However, now that that field has been removed, I can also remove that `if`